package.json

```diff
@@ -40,7 +41,9 @@
     "prop-types": "^15.6.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "react-test-renderer": "^16.2.0"
+    "react-test-renderer": "^16.2.0",
+    "rollup": "^0.55.3",
```

Comment:
i've used rollup as imho its preferable to hide all `build` behind a single CLI tool and command - rollup can output multiple files, even based on multiple configs, in 1 pass